types.tsx

```diff
@@ -9,11 +9,17 @@ export type EpisodeInputStackParamList = {
   CameraFlow: undefined;
 };
 
+/*
+time.toLocaleTimeString("en-US", {
+                  hour: "numeric",
+                  minute: "2-digit",
+                })
+*/
 export type Episode = {
   name: string;
   date: string; // ISO String
-  startTime: number;
-  endTime: number;
+  startTime: string; // Locale String (Ex: 2:30PM, 12:20AM, 7:41PM)
+  endTime: string; // Locale String (Ex: 2:30PM, 12:20AM, 7:41PM)
```

Comment:
Oh this is solid. Just store it as strings, since we don't need to do anything with the dates. I like it!